### PR TITLE
Machine learning

### PR DESCRIPTION
**Pull request by ASLManasa**
<hr>

## Description 📜

ML 6.6: Feedforward Neural Networks (D) #6623

<hr>

## Type of change 📝

- [x ] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [ x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ x] I have performed a self-review of my own code or work.
- [ x] I'm GWOC'21 contributor

### CODE REVIEW

**Comment by ASLManasa** [comment]
fixed changes

File: Machine_Learning/Dimensionality_Reduction_&_Neural_Networks/ML_6.6_Feedforward_Neural_Networks_(D).md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+🔴 **Module** : Dimensionality Reduction & Neural Networks
+
+---
+
+<h1>Neural Networks</h1>
+A Neural network is also known as Artifical Neural Network is a series of algorithms that endeavors to recognize underlying relationships in a set of data through a process that mimics the way bological neurons signal to each other. It is the  In this sense, neural networks refer to systems of neurons, either organic or artificial in nature. These are the heart of the deep learning.
+<br>
+<h1> Feed Forward Neural Network</h1>
+Feed Forward Neural Network is a classification algorithim.It is an artifical neural network in which the nodes doesnt form any cycle or directed acyyclic graph..It is the simplest form of neural network. It opposite is reccurent neural network in which the nodes form a cycle .<br>
```
Comment (Isha307) [inline Machine_Learning/Dimensionality_Reduction_&_Neural_Networks/ML_6.6_Feedforward_Neural_Networks_(D).md:16]: acyyclic -> acyclic 
It -> It's before opposite
and remove one full stop after graph.

File: Machine_Learning/Dimensionality_Reduction_&_Neural_Networks/ML_6.6_Feedforward_Neural_Networks_(D).md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+It forms a single layer perceptron. Structure of a feed forward is an advantage for certain applications.Examples of feed forward neural network is radial basis function networks, which uses activation function.
+<img src="https://images.deepai.org/django-summernote/2019-06-06/5c17d9c2-0ad4-474c-be8d-d6ae9b094e74.png"></img>
+
+<h2>How does a Feed Forward Network work? </h2>
+ It is basically a single layer perceptron. In this model series of input enters the layer and then multipied with its weights. All the values are added to get the sum of weighted inputs and checks if the value is above a specif threshold. If it is then it is set to zero and if less than threshold then it is set to -1.  This model is mostly used in classification tasks.This process of training and learning produces a form of a gradient descent.<br>
```
Comment (Isha307) [inline Machine_Learning/Dimensionality_Reduction_&_Neural_Networks/ML_6.6_Feedforward_Neural_Networks_(D).md:21]: specif -> specific

**Comment by ASLManasa** [comment]
@Isha307 done

**Review by Isha307 — APPROVED** [review]

**Pull request merged**